downloads.md

```diff
@@ -163,6 +163,7 @@ All Flink releases are available via [https://archive.apache.org/dist/flink/](ht
 - Flink 0.6-incubating - 2014-08-26 ([Source](https://archive.apache.org/dist/incubator/flink/flink-0.6-incubating-src.tgz), [Binaries](https://archive.apache.org/dist/incubator/flink/))
 
 ### Flink-shaded
+- Flink-shaded 6.0 - 2019-02-12 ([Source](https://archive.apache.org/dist/flink/flink-shaded-6.0/flink-shaded-6.0-src.tgz))
```

Comment:
Hi @zentol , the `downloads.zh.md` file has been created for Chinese support and not translated yet. So it would be nice to update `downloads.zh.md` too.

Thanks.